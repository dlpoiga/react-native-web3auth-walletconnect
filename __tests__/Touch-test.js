/**
 * @format
 */

import 'react-native';
import React from 'react';
import Touch from '../src/components/common/touch';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';

test('Touch renders correctly', () => {
  const tree = renderer.create(<Touch />).toJSON();
  expect(tree).toMatchSnapshot();
});
