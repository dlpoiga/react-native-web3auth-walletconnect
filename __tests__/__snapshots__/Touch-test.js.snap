// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Touch renders correctly 1`] = `
<View
  accessibilityState={
    Object {
      "disabled": false,
    }
  }
  accessible={true}
  collapsable={false}
  focusable={true}
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "opacity": 1,
    }
  }
/>
`;
